lib/bootscale/entry.rb

```diff
@@ -13,6 +13,7 @@ def initialize(path)
     end
 
     def features
+      return [] unless @path.start_with?('/'.freeze)
```

Comment:
I didn't though of the last one. I guess it's probably the way to go with a warning.